### PR TITLE
IBX-620: Redesigned ezauthor

### PR DESCRIPTION
**Pull request by dew326**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-620
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js [hunk 1]
```diff
@@ -100,6 +100,32 @@
...
+            const container = event.target.closest(SELECTOR_FIELD);
+            const checkboxes = container.querySelectorAll('.ibexa-input--checkbox');
+            const isAnyCheckboxSelected = [...checkboxes].some((checkbox) => checkbox.checked);
+            const bulkDeleteButton = container.querySelector('.ibexa-btn--bulk-remove-author');
+
+            if (isAnyCheckboxSelected) {
+                bulkDeleteButton.removeAttribute('disabled');
+            } else {
+                bulkDeleteButton.setAttribute('disabled', 'disabled');
+            }
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/fieldType/ezauthor.js:113]:
```suggestion
            bulkDeleteButton.toggleAttribute('disabled', !isAnyCheckboxSelected);
```

File: src/bundle/Resources/views/themes/admin/ui/field_type/edit/ezauthor.html.twig [hunk 2]
```diff
@@ -9,36 +9,57 @@
...
+        >
+            <svg class="ibexa-icon ibexa-icon--small ibexa-icon--trash">
+                <use xlink:href="{{ ibexa_icon_path('trash') }}"></use>
+            </svg>
+            <span class="ibexa-btn__label">{{ 'ezauthor.action.delete'|trans|desc('Delete') }}</span>
+        </button>
+    </div>
+    <div class="ibexa-data-source__labels">
+    <label class="ibexa-label form-label">{{ 'ezauthor.name'|trans|desc('Name') }}</label>
+    <label class="ibexa-label form-label">{{ 'ezauthor.Email'|trans|desc('Email') }}</label>
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/field_type/edit/ezauthor.html.twig:37]:
```suggestion
        <label class="ibexa-label form-label">{{ 'ezauthor.name'|trans|desc('Name') }}</label>
        <label class="ibexa-label form-label">{{ 'ezauthor.Email'|trans|desc('Email') }}</label>
```

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Review by lucasOsti — APPROVED** [review]

File: src/bundle/Resources/translations/messages.en.xliff [hunk 3]
```diff
@@ -271,16 +306,46 @@
...
+        <source>Add relations</source>
+        <target state="new">Add relations</target>
+        <note>key: ezobjectrelationlist.add_relation</note>
+      </trans-unit>
+      <trans-unit id="85874e8811087f9c2169915345cdf71d5243182e" resname="ezobjectrelationlist.delete_selected_relations">
+        <source>Delete selected relation(s)</source>
+        <target state="new">Delete selected relation(s)</target>
+        <note>key: ezobjectrelationlist.delete_selected_relations</note>
+      </trans-unit>
+      <trans-unit id="91cee9d11eeed40768d4e5270d91cd5165f4562f" resname="ezobjectrelationlist.limit">
```
Comment (tischsoic) [inline src/bundle/Resources/translations/messages.en.xliff:339]: Were all these translations missing? Do we want to add them all in this PR?

File: src/bundle/Resources/translations/messages.en.xliff [hunk 4]
```diff
@@ -271,16 +306,46 @@
...
+        <source>(max. %limit%)</source>
```
Comment (tischsoic) [inline src/bundle/Resources/translations/messages.en.xliff:340]: Were all these translations missing? Do we want to add them all in this PR?

**Review by tischsoic — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Pull request merged**